src/app/dashboard/news/[id]/NewsDetailClient.js

```
@@ -0,0 +1,391 @@
+                                onEditorStateChange={setShortContentState}
+                                toolbar={editorToolbar}
+                                editorClassName="px-3 min-h-[200px]"
+                                placeholder="Entrez une brève description (visible dans les aperçus)"
+                                wrapperClassName="editor-wrapper"
+                                stripPastedStyles={true}
+                            />
+                        </div>
+                    ) : (
+                        <div 
+                            className="border border-gray-200 p-4 rounded bg-gray-50"
+                            dangerouslySetInnerHTML={{ __html: newsData.shortContent }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)